store/sqlstore/store.go

```diff
@@ -44,6 +52,9 @@ const (
 	// 10.1 would be 100001.
 	// 9.6.3 would be 90603.
 	MinimumRequiredPostgresVersion = 100000
+
+	migrationsDirectionUp   = "up"
+	migrationsDirectionDown = "down"
```

Comment:
We can even make these to be typed constants for extra type safety, and use `migrationDirection` as the type for migrate method. 0/5.

```
type migrationDirection string

const (
 migrationsDirectionUp migrationDirection  = "up"
 migrationsDirectionDown migrationDirection = "down"
)
```

Reply:
sure, 0/5 as well.